frontend/src/scenes/experiments/ExperimentView/ExperimentView.tsx

```diff
@@ -63,6 +64,7 @@ const VariantsTab = (): JSX.Element => {
         <div className="space-y-8">
             <ReleaseConditionsTable />
             <DistributionTable />
+            <CumulativeExposuresChart />
```

Comment:
Please release this behind a feature flag for now. We want to verify that this looks good on production experiments + will need to migrate this to HogQL too (I'll do this).